packages/contracts/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### Added
+
+- Added a `ERC165ContractRegistry` `Component` to register contracts by their address and ERC165 interface ID.
+  
```

Comment:
@mathewmeconry Will the new GitHub workflow be able to handle this correctly?

Reply:
the changelog stuff hasn't been done yet, because I forgot if we want it or not :sweat: 
do we? 
see the sdk repos workflows as an example how it would work

Reply:
We do want it, right @Rekard0 ?

Reply:
yes, please @Michael-A-Heuer